R/conversion.R

```diff
@@ -1,54 +1,42 @@
-# These functions are used in the conversions
-
-#' @param input
+#' Convert tas from K to C
+#'
+#' @param input the matrix of the monthly downscaled tas data
 #' @keywords internal
-#' @return TBD
+#' @return a matrix of tas in degrees C
 
 tas_conversion <- function(input){
 
-  input - 273.15 # K to C
-
+  signif(input - 273.15, digits = 6) # K to C
```

Comment:
@kdorheim For what it's worth, I wouldn't do any rounding in a utility routine like this.  A caller would find it surprising to call a function that purports to convert from K to C, only to find that the result had also been rounded.  If it's absolutely necessary to round here (i.e., the rounding is part of _why_ a user is calling the function),  then the documentation should reflect that.

Reply:
@rplzzz I think I added this because we wanted to control the digits in the pr file to decrease the size of the files saved at the end... but I guess since we are passing in memory we might not need to worry about it any more..